src/utils/validationFunctions.js

```diff
@@ -0,0 +1,21 @@
+import { emailRegex, passwordRegex } from './regexMatchers.js'
+
+export function register(email, password) {
+  if (!email) {
+    throw Error('Email is required')
```

Comment:
The error messages should probably be moved to their own file, perhaps on an object, so we can change them easily if they need to be used in future errors

Reply:
Const's work too, e.g.

`export const ERR_EMAIL_REQUIRED = ''`